app/commands/github/task/destroy.rb

```diff
@@ -0,0 +1,14 @@
+module Github
+  class Task
+    class Destroy
+      include Mandate
+
+      initialize_with :issue_url
+
+      def call
+        task = ::Github::Task.find_by(issue_url: issue_url)
+        task&.destroy
```

Comment:
```suggestion
        ::Github::Task.find_by(issue_url: issue_url)&.destroy
```

Reply:
You could also do `::Github::Task.where(issue_url: issue_url).destroy_all` if you want to avoid the null check. It's effectively the same as we have a unique index on the column. Whichever you prefer is fine by me.

Reply:
I like that.